### PR TITLE
Clarified Introduction Phrasing

### PR DESCRIPTION
**Pull request by 1bardesign**
Just a few slight changes to the introductory paragraph. I got confused while reading through and thought "foundation" was referring to an actual organisation like several other open source foundations. I've clarified the intent of that passage, and hopefully this should avoid any similar future confusion.

I discussed this with user `asonge` on the `#elixir-lang` freenode IRC and they recommended I make a PR, so here it is 🙂

### CODE REVIEW

File: getting-started/introduction.markdown [hunk 1]
```diff
@@ -10,7 +10,7 @@ redirect_from: /getting_started/1.html
 
 Welcome!
 
-In this tutorial, we are going to teach you the Elixir foundation, the language syntax, how to define modules, how to manipulate the characteristics of common data structures, and more. This chapter will focus on ensuring Elixir is installed and that you can successfully run Elixir's Interactive Shell, called IEx.
+In this tutorial, we are going to teach you about Elixir - the language syntax, how to define modules, how to manipulate the characteristics of common data structures, and more. This chapter will focus on ensuring that Elixir is installed and that you can successfully run Elixir's Interactive Shell, called IEx.
```
Comment (whatyouhide) [inline getting-started/introduction.markdown]: Maybe "the fundamentals" of Elixir?

Comment (1bardesign) [reply getting-started/introduction.markdown]: That'd definitely be an option, but I think simpler is better when it comes to something accessibility-sensitive like the intro tutorial. If others would prefer that wording or any other I'm happy to update the PR tomorrow though 👍

Comment (eksperimental) [reply getting-started/introduction.markdown]: "the building blocks of Elixir" could be another option

**Comment by josevalim** [comment]
:heart: :green_heart: :blue_heart: :yellow_heart: :purple_heart:

**Pull request merged**